crates/biome_css_formatter/tests/specs/scss/expression/map-closing-comments.scss

```
@@ -0,0 +1,16 @@
+$foo:('property1':(),'property2':1,/** comment 1 */
+/* comment 2 */
+);
+
+$bar:('property1':(),'property2':1,
+/** comment 1 */
+/* comment 2 */
+);
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)